include/constants/battle_config.h

```diff
@@ -133,6 +133,7 @@
 #define B_PARALYZE_ELECTRIC         GEN_6 // In Gen6+, Electric-type Pokémon can't be paralyzed.
 #define B_POWDER_GRASS              GEN_6 // In Gen6+, Grass-type Pokémon are immune to powder and spore moves.
 #define B_STEEL_RESISTANCES         GEN_6 // In Gen6+, Steel-type Pokémon are no longer resistant to Dark and Ghost moves.
+#define B_TRAINERS_STEAL_ITEMS      TRUE  // If TRUE, trainer with thief/pickpocket will temporarily steal your items
```

Comment:
I think this should be reversed, as "Trainers steal items" could mean that they permanetly steal them.

Reply:
reversed as in FALSE by default? I can change it to `B_TRAINERS_KNOCK_OFF_ITEMS` too, since that doesn't imply a permanent loss.

Reply:
Changing the name of the config should help, but I feel it's not clear enough that the vanilla effect is of permanent loss.
That's why I suggested keeping the current config name but reversing the checks themselves.